public/stylesheets/site/2.0/22-system-messages.css

```diff
@@ -51,7 +51,7 @@ div.error {
     box-shadow: 1px 1px 2px;
 }
 
-.notes {
+p.notes {
```

Comment:
The notes field on the posting form was playing up because of a clear meant for notes in system messages (which are [either paragraphs or unordered lists](https://github.com/search?q=repo%3Aotwcode%2Fotwarchive+notes+path%3A%2F%5Eapp%5C%2Fviews%5C%2F%2F&type=code)), but the notes elements _also_ had the notes class. You'll see `ul.notes` already has this clear above, so we just needed to make sure it also applied to paragraphs with the `notes` class.

Fixing this allowed me to remove the clear from `.blurb blockquote`, which had the notes class in the bookmark blurb. https://github.com/otwcode/otwarchive/blob/f21f823d4842860e921909f6043cec3d609271b8/app/views/bookmarks/_bookmark_user_module.html.erb#L50